Runtime/SimpleAddress.cs

```diff
@@ -3,6 +3,7 @@
 
 namespace PassivePicasso.SimplyAddress
 {
+    [ExecuteAlways]
```

Comment:
This has no effect as SimpleAddress doesn't execute any code and ExecuteAlways is not applied to descendant types